code/src/components/InputCheckbox.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../Styles/InputCheckbox.css';
+
+const InputCheckbox = ({ question, array, selectedMonths, onMonthsChange }) => {
+	return (
+		<>
+			<h2 tabIndex="0">{question}</h2>
+			<div
+				className="checkbox-container"
+				aria-label="Check preferred months to be able to proceed the survey"
+			>
+				{array.map(item => (
```

Comment:
Maybe the naming of this array could be a bit more informative as to what it contains.

Reply:
Thank you for the input, I have updated the naming :)